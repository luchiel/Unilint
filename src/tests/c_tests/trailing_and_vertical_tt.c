int main()
{
    int a, b;
    scanf("%d%d", &a, &b);
    switch(a)
    {
        case 1:   printf("too small"); break;
        case 100: printf("too big");  break;    
        case 45: print("gorgeous!");  break;
    }
}