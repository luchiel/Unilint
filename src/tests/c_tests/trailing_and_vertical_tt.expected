7:16: more than one whitespace
8:37: more than one whitespace
8:45: trailing whitespaces
9:37: more than one whitespace
